js-jasmine/src/aged_brie.js

```diff
@@ -0,0 +1,22 @@
+class AgedBrie {
+    constructor(items = []) {
+    this.items = items
+    }
```

Comment:
Your code looks really good and is easy to follow, I like that you refactored it into separate classes for each item, it makes it really easy to understand what each class is doing.